msteams-platform/resources/teams-updates.md

```diff
@@ -0,0 +1,59 @@
+ms.topic: reference
+---
+# Microsoft Teams update
+
+[The new Microsoft Teams client](https://www.microsoft.com/en-us/microsoft-365/blog/2023/03/27/welcome-to-the-new-era-of-microsoft-teams/) is reimagined from the ground up with performance in mind. It's faster, simpler, smarter, and flexible to provide better experience for your apps and users. The new Teams client supports all the existing Teams app capabilities except Adaptive Card tabs. If you have an app that runs inside the Classic Teams, the app will most likely run in the new Teams client without any issues.
+
+The following are the advantages the new Teams client:  
+
+* The new Teams client uses the Evergreen version of Microsoft Edge WebView2 to ensure Teams client is always up to date with the latest fixes and improvements available in Microsoft Edge and Chromium.
+
+* The new Teams client has been rebuilt from the ground up with performance in mind and includes all the platform infrastructure responsible for bootstrapping your app and powering the SDK APIs that it uses.  
+
+You can use the following property to identify your app usage in the new Teams or Classic Teams client:
+
+* For TeamsJS v1.x: [`hostName`](/javascript/api/@microsoft/teams-js/hostname?view=msteams-client-js-latest&preserve-view=true)
+* For TeamsJS v2.x: [`AppHostInfo`](/javascript/api/@microsoft/teams-js/app.appinfo?view=msteams-client-js-latest&preserve-view=true#@microsoft-teams-js-app-appinfo-host)
+
+The new Teams or Classic Teams client are represented using the `teams` and `teamsModern` fields, respectively.
+
+## Timelines and rollout
+
+To ensure a smooth transition, a phased rollout of the new platform is planned as follows:
+
+* **Developer Preview**: The new Teams client is available in Public Developer Preview starting June 2023. You can access the new platform and test your apps. We encourage you to adopt the feature early and provide feedback to help refine the platform.
+
+* **Availability of all platform features from Classic Teams**: All the platform features from Teams classic will be available in the new Teams client by August 2023.
+
+## Known issues
+
+> [!NOTE]
+>
+> * We recommend to test apps, tabs, messaging extensions, bots, and link unfurling after switching from the Classic Teams client to the new Teams client.
+> * Adaptive Card tabs arn't supported in the new Teams client. If your app is using Adaptive Card tabs, we recommend to rebuild the tab as a web-based tab. For more information, see [Build tabs for Teams](../tabs/how-to/build-adaptive-card-tabs.md).
+
+**Teams features that will be supported soon**
+
+* [Share in Teams](../concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md).
+
+* APIs used in Teams that require [device permissions](../concepts/device-capabilities/native-device-permissions.md).
+
+* If you have a [meeting extension](../apps-in-teams-meetings/teams-apps-in-meetings.md) app, the app might not support all the meeting extensibility features.
```

Comment:
This can be removed since we've already added support for all meeting app capabilities.

Reply:
removed the issue.